doc/install/build_install.rst

```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
         git submodule update
     Or download submodules recursively when checking out:
         git clone --recurse-submodules https://github.com/arbor-sim/arbor.git
+
+
+Build in conda virtual environment using miniconda
+==================================================
+
+If you hope to install Arbor from source in a virtual environment in order not to interfere with other Arbor versions or software you have installed, or if you want to use features that are not included in the official release, you can use miniconda for this purpose. Otherwise, type pip install arbor in your terminal to install arbor. But you will have to fix two issues, one is feeding the correct Python path, and the other is 'GLIBCXX 3.4.30 not found in conda environment'.
+
+.. code-block:: bash
+
+#create a virtual environment
+
+conda create --name arbor_test
+conda activate arbor_test
+
+
+#go to the folder and clone the Arbor source package from GitHub
+
+cd ~/miniconda3/envs/arbor_test/
+mkdir src
+cd src
+git clone https://github.com/arbor-sim/arbor.git --recurse-submodules
+
+
+#install python and numpy in this environment
+
+conda install python=3.12.2
+conda install numpy
+
+
+#start the build
+
+cd arbor
+mkdir build
+cd build
+
+cmake .. -GNinja -DCMAKE_CXX_COMPILER=$(which g++) -DCMAKE_C_COMPILER=$(which gcc) -DARB_WITH_PYTHON=ON -DARB_VECTORIZE=ON -DPython3_EXECUTABLE=$(which python3) -DARB_USE_BUNDLED_LIBS=ON
+
+
+#activate ninja to install
+
+ninja
+sudo ninja install
```

Comment:
You don't need this and it'll install to the full system, i.e. outside any conda env, so maybe not recommend it ;)